app/controllers/api/api_controller.rb

```diff
@@ -17,7 +17,8 @@ class ApiController < ApplicationController
       Subjects::Selector::MissingSubjectSet,
       Subjects::Selector::MissingSubjects,                 with: :not_found
     rescue_from ActiveRecord::RecordInvalid,               with: :invalid_record
-    rescue_from Api::LiveProjectChanges,                   with: :forbidden
+    rescue_from Api::LiveProjectChanges,
+      Api::ExportDisabled,                                 with: :forbidden
```

Comment:
What happens on the front end when someone hits the button and expects to get an export? If we turn these off we should have a way of letting PFE as well as people who are hitting the API directly know that they're not gonna get anything.